### PR TITLE
Show all subject and edit, delete

### PR DESCRIPTION
**Pull request by limkimhuor**

### CODE REVIEW

File: app/controllers/supervisor/subjects_controller.rb [hunk 1]
```diff
@@ -7,12 +7,41 @@ def create
...
+    @subject = Subject.find params[:id]
+  end
+
+  def edit
+    @subject = Subject.find params[:id]
+  end
+
+  def update
+    @subject = Subject.find params[:id]
+    if @subject.update_attributes(subject_params)
```
Comment (RathanakSreang) [inline app/controllers/supervisor/subjects_controller.rb]: Remove `()`

File: app/views/supervisor/subjects/_subject.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<ul class = "subjects">
+  <li>
+    <%= link_to subject.name, [:supervisor, subject] %>
+    <span class="timestamp">
+      created: <%= time_ago_in_words subject.created_at %> ago.
+    </span>
+    <%= link_to "edit", edit_supervisor_subject_path(subject) %>
+    <%= link_to "delete", [:supervisor, subject], 
+                            method: :delete, data: { confirm: "You sure?" } %>
```
Comment (RathanakSreang) [inline app/views/supervisor/subjects/_subject.html.erb]: move to en.yml

**Comment by RathanakSreang** [comment]
Ok

File: app/views/supervisor/subjects/new.html.erb [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <% provide :title, "Create new subject" %>
-<h1>Create new subject</h1>
-
+  <h1><%= t(:title, title: :"Create new subject") %></h1>
```
Comment (dieunb) [inline app/views/supervisor/subjects/new.html.erb]: indent

File: app/views/supervisor/subjects/_subject.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<ul class = "subjects">
+  <li>
+    <%= link_to subject.name, [:supervisor, subject] %>
+    <span class="timestamp">
+      created: <%= time_ago_in_words subject.created_at %> ago.
+    </span>
+    <%= link_to t("links.edit"), edit_supervisor_subject_path(subject) %>
+    <%= link_to t("links.delete"), [:supervisor, subject], 
+                            method: :delete, data: { confirm: "You sure?" } %>
```
Comment (dieunb) [inline app/views/supervisor/subjects/_subject.html.erb]: `{confirm: "You sure?"}`

**Comment by samnangcattor** [comment]
ok

**Comment by RathanakSreang** [comment]
Ok

File: app/controllers/supervisor/subjects_controller.rb [hunk 5]
```diff
@@ -1,4 +1,13 @@
 class Supervisor::SubjectsController < ApplicationController
+  before_action :set_subject, only: [:show, :edit, :update, :destroy]
```
Comment (dieunb) [inline app/controllers/supervisor/subjects_controller.rb]: Nên dùng `except` cho ngắn gọn

**Pull request merged**